### PR TITLE
MacOS tests updates

### PR DESCRIPTION
**Pull request by zemtsov-artem**

### CODE REVIEW

**Comment by zemtsov-artem** [comment]
Is there any blocker to merge it?

File: .travis.yml [hunk 1]
```diff
@@ -44,6 +48,7 @@ before_install:
 install:
 
   # macOS dev dependencies:
+  - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then sudo sqlite3 "/Library/Application Support/com.apple.TCC/TCC.db" "INSERT INTO access VALUES ('kTCCServiceAccessibility','com.apple.dt.Xcode-Helper',0,1,1,NULL,NULL);"; fi
```
Comment (zemtsov-artem) [inline .travis.yml]: Remove changes

File: pywinauto/macos/macos_functions.py [hunk 2]
```diff
@@ -99,6 +119,7 @@ def get_instance_of_app(name):
             return app
     if (is_debug):
         print ("App is not allready running.Can't get instance")
```
Comment (vasily-v-ryabov) [inline pywinauto/macos/macos_functions.py]: `allready` -> `already`

File: pywinauto/macos/macos_functions.py [hunk 3]
```diff
@@ -71,6 +72,24 @@ def launch_application_by_bundle(bundle_id, new_instance=True):
...
+        param = NSWorkspaceLaunchNewInstance
+    else:
+        param = NSWorkspaceLaunchAllowingClassicStartup
+
+    r = get_ws_instance().launchApplicationAtURL_options_configuration_error_(url,
+            param,
+            {},
+            None)
+    if not r[0]:
+            raise RuntimeError('Error launching specified application. Result: {}'.format(r))
```
Comment (vasily-v-ryabov) [inline pywinauto/macos/macos_functions.py]: Too long indent

File: pywinauto/macos/macos_functions.py [hunk 4]
```diff
@@ -71,6 +72,24 @@ def launch_application_by_bundle(bundle_id, new_instance=True):
...
+
+def launch_application_by_url(url, new_instance=True):
+    if (new_instance):
+        param = NSWorkspaceLaunchNewInstance
+    else:
+        param = NSWorkspaceLaunchAllowingClassicStartup
+
+    r = get_ws_instance().launchApplicationAtURL_options_configuration_error_(url,
+            param,
+            {},
```
Comment (vasily-v-ryabov) [inline pywinauto/macos/macos_functions.py]: Word wrap seems not necessary, it should fit into 120 symbols.

File: pywinauto/macos/macos_functions.py [hunk 5]
```diff
@@ -42,6 +42,7 @@
 import subprocess
 import os
 from ApplicationServices import *
```
Comment (vasily-v-ryabov) [inline pywinauto/macos/macos_functions.py]: It would be nice to eliminate wildcard import.

File: pywinauto/macos/macos_functions.py [hunk 6]
```diff
@@ -42,6 +42,7 @@
 import subprocess
 import os
```
Comment (vasily-v-ryabov) [inline pywinauto/macos/macos_functions.py]: Built-in modules should be imported first: os, subprocess. Then 3-rd party modules (six, all PyObjC stuff), then project specific modules. Grouped imports should be separated by empty line.

**Review by vasily-v-ryabov — CHANGES_REQUESTED** [review]

**Comment by zemtsov-artem** [comment]
All tests are passed.
Fixed code style as well.

**Review by vasily-v-ryabov — APPROVED** [review]

**Pull request merged**